### PR TITLE
bootstrapping graphql server

### PR DESCRIPTION
**Pull request by mfrohberg**
`/api/graphql` endpoint working...

### CODE REVIEW

File: pages/api/graphql.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const apolloServer = new ApolloServer({ typeDefs, resolvers })
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+}
+
+export default apolloServer.createHandler({ path: '/api/graphql' })
```
Comment (aretecode) [inline pages/api/graphql.ts:17]: this is super nifty! 😎 
https://www.apollographql.com/docs/apollo-server/v1/servers/micro/

File: pages/api/graphql.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { ApolloServer, gql } from 'apollo-server-micro'
+
+const typeDefs = gql`
+  type Query {
```
Comment (aretecode) [inline pages/api/graphql.ts]: once it has correct queries will merge

**Review by aretecode — CHANGES_REQUESTED** [review]

File: pages/api/graphql.ts (hunk 1 above)
Comment (mfrohberg) [reply pages/api/graphql.ts:17]: Yeah it is pretty cool. The next integration makes it hot-reloadable

File: src/graphql/schema.gql.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  type User {
+    uid: ID!
+    name: String
+    email: String
+    picture: String
+    nickname: String
+    adviceList: [Advice]
+    familyMembers: [FamilyMember]
```
Comment (aretecode) [inline src/graphql/schema.gql.ts:30]: should be `familyList`

File: src/constants.ts [hunk 4]
```diff
@@ -76,3 +76,43 @@ export const DEFAULT_FAMILY_MEMBER_LIST: FamilyMemberItemType[] = [
...
   },
 ]
+
+export const DEFAULT_USER: any = {
+  uid: -1,
+  name: 'Testy Jones',
+  email: 'testy.jones@gmail.com',
+  picture: 'https://s.gravatar.com/avatar/a3c1eb95d1396a1b878752e9cdd9613a?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Fmi.png',
+  nickname: 'testy.jones',
+  adviceList: [
```
Comment (aretecode) [inline src/constants.ts:86]: can use the default list

File: src/constants.ts [hunk 5]
```diff
@@ -76,3 +76,43 @@ export const DEFAULT_FAMILY_MEMBER_LIST: FamilyMemberItemType[] = [
...
+    {
+      uid: 3,
+      familyMemberId: -3,
+      title: 'Rocky Balboa',
+      text: 'When life gets hard',
+      description: ' Son, nobody will hit you as hard as life. But...',
+      happenedOn: '01/08/2015',
+    },
+  ],
+  familyMembers: [
```
Comment (aretecode) [inline src/constants.ts:104]: should be `familyList`

File: pages/api/graphql.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { ApolloServer } from 'apollo-server-micro'
+import {
+  typeDefs,
+  resolvers,
+}  from '../../src/graphql'
+
+const apolloServer = new ApolloServer({
```
Comment (aretecode) [inline pages/api/graphql.ts:7]: nice and simple 👍

File: src/graphql/apolloClient.ts [hunk 7]
```diff
@@ -96,9 +96,9 @@ export function createInstance(
     console.info(`starting request for ${operation.operationName}`)
 
     if (forward !== undefined) {
-      return forward(operation).map(data => {
+      return forward(operation).map(datum => {
```
Comment (aretecode) [inline src/graphql/apolloClient.ts:99]: Oh, is this Channing?

File: src/constants.ts [hunk 8]
```diff
@@ -76,3 +76,43 @@ export const DEFAULT_FAMILY_MEMBER_LIST: FamilyMemberItemType[] = [
     name: 'Barbara',
   },
 ]
+
+export const DEFAULT_USER: any = {
```
Comment (aretecode) [inline src/constants.ts:80]: should have typos for the user (_I've added now_)

**Review by aretecode — COMMENTED** [review]
looks good to me (typings & typo issue fixed here https://github.com/aretecode/konsilos/commit/bf96ec0cefc2f093859d5c1b20ee276986c5f9da)

**Review by aretecode — APPROVED** [review]

**Pull request merged**